pymeasure/display/manager.py

```diff
@@ -279,3 +278,78 @@ def abort(self):
             self._worker.stop()
 
             self.aborted.emit(self._running_experiment)
+
+# JM: ####################################################################
```

Comment:
This comment should be removed before merge.

Reply:
It looks like a number of `# JM:` comments are still in the PR. Can you remove them?
```
$ grep -rnI "# JM"
pymeasure/display/widgets.py:263:        self.x_axis_changed.emit(x_axis) # JM: Not sure if this does anything now...
pymeasure/display/widgets.py:267:        self.y_axis_changed.emit(y_axis) # JM: Not sure if this does anything now...
pymeasure/display/widgets.py:279:        self.plot_widget = pg.PlotWidget(self, background='#ffffff') # JM: NOTE: This is just pulling a base pg PlotWidget, so no infinite regress!
pymeasure/display/widgets.py:403:    # JM: Maybe should have a change image function, so that we know there is 
pymeasure/display/manager.py:282:# JM: ####################################################################
pymeasure/display/curves.py:94:        # JM: NOTE: Below we're making an assumption on the form of *any* procedure 
pymeasure/display/curves.py:101:        self.xsize = int(np.ceil((self.xend - self.xstart) / self.xstep)) + 1 # JM: add one since arange doesn't include bound, but maybe experiment will give it. Just defaults to empty pixels, not the end of the world.
pymeasure/display/curves.py:105:        self.ysize = int(np.ceil((self.yend - self.ystart) / self.ystep)) + 1 # JM: add one since arange doesn't include bound, but maybe experiment will give it. Just defaults to empty pixels, not the end of the world.
pymeasure/display/windows.py:31:from .curves import ResultsCurve # JM: only need to import another if we want a PlotterWindow equivalent
pymeasure/display/windows.py:501:# JM: ############################################################################
pymeasure/display/windows.py:645:    def browser_item_changed(self, item, column): # JM: adds or removes item based on checkbox in browser item. Need to destroy old and add new.
pymeasure/display/windows.py:680:            action_change_color = QtGui.QAction(menu) # JM: probably remove this??
pymeasure/display/windows.py:772:        # if color is None: # JM: I guess we could extend this to colormaps?
pymeasure/display/windows.py:825:    def setup_plot(self, plot): # JM: Will need to change this probably.
pymeasure/display/windows.py:839:    def setup_im_plot(self, im_plot): # JM: Will need to change this probably.

```